ui/pages/lectures/[id].tsx

```diff
@@ -103,15 +104,13 @@ export default Lecture;
 export async function getStaticProps(context) {
   let lectures;
   try {
-    lectures = await getApiRoot().get(`/lectures/${context.params.id}/`);
+    lectures = await axios.get(
+      `http://localhost:3000/api/lectures/${context.params.id}/`
```

Comment:
I think it might be more correct to add it as `/api/lectures....`.

I think i saw somewhere in the docs that this will automatically target the current nextjs api

Reply:
I tried but doing that there is no port specified. Because we are running the port in port `3000` and the default being `80`, it doesn't work as expected